backend/src/packages/users/user.service.ts

```diff
@@ -35,6 +35,8 @@ class UserService implements IService {
         email: payload.email,
         passwordSalt: 'SALT', // TODO
         passwordHash: 'HASH', // TODO
+        firstName: payload.firstName,
+        lastName: payload.lastName,
```

Comment:
If `UserSignUpRequestDto` was not changed - I'm not sure that this props exist on current type

Reply:
This dto is already in branch development